File: event/src/test/java/de/danoeh/antennapod/event/SyncServiceEventTest.java
package de.danoeh.antennapod.event;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class SyncServiceEventTest {

    @Test
    public void messageResIdIsStored() {
        SyncServiceEvent event = new SyncServiceEvent(42);
        assertEquals(42, event.getMessageResId());
    }
}

File: event/src/test/java/de/danoeh/antennapod/event/SyncServiceEventTest.kt
package de.danoeh.antennapod.event

import org.junit.Assert.assertEquals
import org.junit.Test

class SyncServiceEventTest {

    @Test
    fun messageResIdIsStored() {
        val event = SyncServiceEvent(42)
        assertEquals(42, event.messageResId)
    }
}
